### DISCUSSION TITLE
Doc comment on Environment variables (/config-refs/env-vars)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1231] [Docs/blog comments]
# Doc comment on Environment variables (/config-refs/env-vars)

Overview

https://weaviate.io/developers/weaviate/config-refs/env-vars

**Comment by TamHHM** [comment]
Should provide all possible choices instead of just examples or at least link the the pages that provide all choices!

**Reply by rakesh-k** [reply]
I cannot agree more. The documentation needs clarity and completeness.

**Comment by josevonroth** [comment]
In a weviate vector database that is hosted on an EC2 instance, how or where can I specify how many CPUs can or should be used by the database? 
Or does a weviate database always use the full amount of CPUs?

**Reply by dudanogueira** [reply]
Hi! Weaviate automatically handles parallelization to ensure that it does not exceed the number of CPU cores. If you want to limit the available CPUs, you will need to do it from docker or kubernetes.

**Comment by sri-postgres-cassandra** [comment]
Can someone please give me the location where does the weaviate error logs get saved ?

**Reply by dudanogueira** [reply]
Hi! Weaviate will have it's logs at stdout. So it will depend on how you have deployed Weaviate. For example, if using docker, you can check the logs like so:

```
docker logs <container_name_or_id>
```

For a Kubernetes deployment, you would use the kubectl command to access logs:

```
kubectl logs <pod_name> -n <namespace>
```
